### ISSUE TITLE
[Bug]: Search doesn't respect disablePathToLower option and returns case sensitive URLs

### ISSUE THREAD

**Issue by rudelm** [CaiJimmy/hugo-theme-stack#778]
### What happened?

I'm not sure when this happened, but recently when I searched on my blog, I was able to open the search results. This stopped or never worked, I'm not sure about it.

Example:
Search for mqtt: https://centurio.net/page/search/?keyword=mqtt
Search results are case sensitive, clicking on one returns a 404

If I go back to the found pages from the frontpage, I'll see correct case insensitive links which are all lower case:
https://centurio.net/2019/12/16/configure-mosquitto-mqtt-broker-user-authentication-in-docker-running-on-synology-nas/

That corresponds to the folder structure on the server, which is all lower case.

my config.yaml:
```yaml
baseURL: https://centurio.net
languageCode: en-us
title: Centurios Blog
theme: "stack"
enableGitInfo: true

frontmatter:
  lastmod: ['lastmod', ':git', 'date', 'publishDate']

markup:
  tableOfContents:
    endLevel: 4
    ordered: false
    startLevel: 1

  disablePathToLower: false
permalinks:
  post: "/:year/:month/:day/:title/"

params:
  rssFullContent: true
  article:
    toc: true
    readingTime: true
  comments:
    enabled: false
  sidebar:
    subtitle: Macs, Linux, IoT and Photography
    avatar:
      enabled: true
      local: true
      src: img/avatar.png
  widgets:
    enabled:
      - search
      - categories
      - archives
      - tag-cloud
    archives:
      limit: 5
    tagCloud:
      limit: 10
    categoriesCloud:
      limit: 20

menu:
    social:
        - identifier: github
          name: GitHub
          url: 'https://github.com/rudelm/'
          params:
            icon: github
            newTab: true
        
        - identifier: mastodon
          name: Mastodon
          url: 'https://infosec.exchange/@rudelm'
          params:
            icon: mastodon
            newTab: true

        - identifier: twitter
          name: Twitter
          url: 'https://twitter.com/rudelm'
          params:
            icon: twitter
            newTab: true
```

### Hugo version

v0.110.0+extended darwin/arm64

### Theme version

3.16.0

### What browsers are you seeing the problem on?

Chrome

### More information about the browser

109.0.5414.119 macOS Ventura 13.2

### Link to the page with bug

https://centurio.net/page/search/?keyword=mqtt

**Comment by CaiJimmy** [comment]
> Example:
> Search for mqtt: https://centurio.net/page/search/?keyword=mqtt
> Search results are case sensitive, clicking on one returns a 404

Is it fixed already? Because I'm not able to reproduce it. 

This should not be related to the theme anyway I think, it uses the standard Hugo parameter `.Permalink` to output the permalink.

**Comment by rudelm** [comment]
yes, sorry I forgot to update. I've removed in the config.yaml `disablePathToLower: true` as I already hat this set to false in the `markup` section and did a complete reupload of the public folder via FTP. Looks like it worked this time. When I wrote the ticket, I've already made the change to the file (as it is in the first post) but uploaded only the changes.

That seems to have at least fixed it now to make everything lower case which is sufficient for me. I stopped worrying about any broken links by now and just want it all consistent.

Thanks for your response and work on the theme, much appreciated :) will reopen the ticket when I see another missbehaving link.
